pkg/sql/opaque.go

```diff
@@ -228,6 +230,7 @@ func init() {
 		&tree.Grant{},
 		&tree.GrantRole{},
 		&tree.ReassignOwnedBy{},
+		&tree.DropOwnedBy{},
```

Comment:
nit: alphabetical order

Reply:
Will change! Thanks, didn't realize it was alphabetical.